src/rsz/include/rsz/Resizer.hh

```diff
@@ -42,6 +42,7 @@
 #include "db_sta/dbSta.hh"
 #include "sta/UnorderedSet.hh"
 #include "sta/Path.hh"
+#include "gui/gui.h"
```

Comment:
Why is this needed in the public header?

Reply:
Switched to a forward decl of Gui class.